records_mover/db/bigquery/bigquery_db_driver.py

```diff
@@ -24,70 +22,14 @@ def __init__(self,
-        """Loads the data from the RecordsDirectory instance named
-        'directory'.  Guarantees a manifest file named 'manifest' is
-        written to the target directory pointing to the target
-        records.
-
-        Returns number of rows loaded (if database provides that
-        info).
-
-        """
-        return self._bigquery_loader.load(schema=schema,
-                                          table=table,
-                                          load_plan=load_plan,
-                                          directory=directory)
-
-    def load_from_fileobj(self, schema: str, table: str,
-                          load_plan: RecordsLoadPlan, fileobj: IO[bytes]) -> Optional[int]:
-        return self._bigquery_loader.load_from_fileobj(schema=schema,
-                                                       table=table,
-                                                       load_plan=load_plan,
-                                                       fileobj=fileobj)
-
-    def can_load_from_fileobjs(self) -> bool:
-        return True
-
-    def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool:
-        return self._bigquery_loader.can_load_this_format(source_records_format)
-
-    def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
-        return self._bigquery_loader.known_supported_records_formats_for_load()
-
-    def unload(self,
-               schema: str,
-               table: str,
-               unload_plan: RecordsUnloadPlan,
-               directory: RecordsDirectory) -> int:
-        """Writes table specified to the RecordsDirectory instance named 'directory'
-        per the UnloadPlan named 'unload_plan'.  Guarantees a manifest
-        file named 'manifest' is written to the target directory pointing
-        to the target records.  After ensuring other records-related
-        metadata is provided, you can call directory.finalize_manifest()
-        to move this to the final location of _manifest, signaling to
-        readers that the Records directory is finalized.
-
-        Returns number of rows unloaded.
-        """
-        raise NotImplementedError(f"unload not implemented for this database type")
-
-    def can_unload_this_format(self, target_records_format: BaseRecordsFormat) -> bool:
-        return False
-
-    def known_supported_records_formats_for_unload(self) -> List[BaseRecordsFormat]:
-        return []
-
-    def best_records_format_variant(self, records_format_type: RecordsFormatType) ->\
-            Optional[str]:
-        if records_format_type == 'delimited':
-            return 'bigquery'
-        else:
-            return None
+    def loader(self) -> Optional[LoaderFromRecordsDirectory]:
+        return self._bigquery_loader
+
+    def loader_from_fileobj(self) -> LoaderFromFileobj:
+        return self._bigquery_loader
+
+    def unloader(self) -> None:
+        return None
```

Comment:
Now we have a few interfaces to implement and return instances of - making the loader/unloader class a first class citizen in the driver API.